packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -170,18 +170,23 @@ export class WaferMap extends FoundationElement {
             this.eventCoordinator.detachEvents();
             if (this.waferMapUpdateTracker.requiresContainerDimensionsUpdate) {
                 this.dataManager.updateContainerDimensions();
+                this.renderer.updateSortedDies();
```

Comment:
If I'm reading this right all the code paths are still resulting in drawWafer being called?

If that's the case I'd recommend moving the drawWafer call out of updateSortedDies and keep it where it was at the end of this function running in all code paths.

An alternative could be renaming the call to `updateSortedDiesAndDrawWafer`.  As just the name `updateSortedDies` it seems to be hiding the important two responsibilities of updating the dies sort order and being responsible for calling drawWafer.